src/Api/Controllers/OrganizationsController.cs

```diff
@@ -163,10 +163,17 @@ public async Task<OrganizationResponseModel> Post([FromBody]OrganizationCreateRe
             }
 
             var policies = await _policyRepository.GetManyByUserIdAsync(user.Id);
-            if (policies.Any(policy => policy.Enabled && policy.Type == PolicyType.SingleOrg))
+            foreach (var policy in policies)
```

Comment:
Move your test logic up here into the `foreach`, `foreach (var policy in policies.Where(p => p.Enabled && p.Type == PolicyType.SingleOrg))`